### PR TITLE
update SISNR doc to make it clear we display minus the SISNR

### PR DESCRIPTION
**Pull request by adefossez**
we report minus the SISNR so that it can also be used as a training loss, but that can be confusing !

### CODE REVIEW

**Review by JadeCopet — APPROVED** [review]
Thanks!

File: audiocraft/losses/sisnr.py [hunk 1]
```diff
@@ -41,6 +41,9 @@ class SISNR(nn.Module):
 
     Input should be [B, C, T], output is scalar.
 
+    ..Warning:: We report here minus SISNR, so negative values are better. This is so that
+        this can be naturally used as a loss!
+
```
Comment (0xlws) [inline audiocraft/losses/sisnr.py:46]: @adefossez  i kindly request to update the following warning messages, these updates are based on my understanding and aim to avoid confusion when reading the documentation. i apologize for any inconvenience caused by my lack of understanding. thank you for considering my request. 

eg: ..Warning:: The reported values presented here represent the negation of SISNR. Consequently, negative values indicate better results. This format is intentionally designed to facilitate the convenient usage of SISNR as a loss function.

File: docs/METRICS.md [hunk 2]
```diff
@@ -16,6 +16,8 @@ We provide an implementation of the Scale-Invariant Signal-to-Noise Ratio in PyT
 No specific requirement is needed for this metric. Please activate the metric at the
 evaluation stage with the appropriate flag:
 
+**Warning:** As we also allow SISNR to be used as a training loss, we actually report minus the SISNR, e.g. negative values are best.
+
```
Comment (0xlws) [inline docs/METRICS.md:20]: continued:
**Warning** Please be aware that in the case of using SISNR as a training loss, the reported values are represented as negative. This implies that the best results are indicated by lower (negative) values.

assuming they do not change the original intention or message.

**Review by 0xlws — CHANGES_REQUESTED** [review]

File: audiocraft/losses/sisnr.py (hunk 1 above)
Comment (adefossez) [reply audiocraft/losses/sisnr.py:47]: i have extended a bit the warnings
